test/README.md

```diff
@@ -45,54 +47,54 @@ set PYTHONUTF8=1
 Individual tests can be run by directly calling the test script, e.g.:
 
 ```
-test/functional/feature_rbf.py
+build/test/functional/feature_rbf.py
 ```
 
 or can be run through the test_runner harness, eg:
 
 ```
-test/functional/test_runner.py feature_rbf.py
+build/test/functional/test_runner.py feature_rbf.py
 ```
 
 You can run any combination (incl. duplicates) of tests by calling:
 
 ```
-test/functional/test_runner.py <testname1> <testname2> <testname3> ...
+build/test/functional/test_runner.py <testname1> <testname2> <testname3> ...
 ```
 
 Wildcard test names can be passed, if the paths are coherent and the test runner
 is called from a `bash` shell or similar that does the globbing. For example,
 to run all the wallet tests:
 
 ```
-test/functional/test_runner.py test/functional/wallet*
-functional/test_runner.py functional/wallet* (called from the test/ directory)
-test_runner.py wallet* (called from the test/functional/ directory)
+build/test/functional/test_runner.py test/functional/wallet*
+functional/test_runner.py functional/wallet*  # (called from the build/test/ directory)
```

Comment:
nit: the command line usually doesn't interpret these as comments

Reply:
Bash does doesn't it?

Reply:
The latest version seems to handle it indeed:

> bash-3.2$ echo 12 #34
 12

but with zsh it's
> echo 12 #34
12 #34

Reply:
The same Bash style for comments is used in other docs. So, it can be improved in a separated PR.